typings.d.ts

```diff
@@ -1,5 +1,10 @@
 /// <reference path="./node_modules/typed-graphql/graphql.d.ts" />
 
+declare namespace NodeJS {
```

Comment:
Oh, OK; is this a browserify thing? I guess I wonder if there are some defined typings for it. Probably not a big deal to include this though.